tests/performance/index.js

```diff
@@ -56,7 +56,7 @@ if (startNow) {
   if (!process.browser) {
     // the two strings are to fool Browserify, because this test
     // fails in Node 0.11-0.12 due to sqlite3 being incompatible
-    global.PouchDB.plugin(require('../../packages/node_modules/' +
+    PouchDB.plugin(require('../../packages/node_modules/' +
```

Comment:
I thought it might be something that you missed before in the cleanup, I couldn’t find it out myself